### PR TITLE
Implement Tap feature in creator hub 2.0

### PR DESCRIPTION
**Pull request by ahmadrns**
Implement tapping of NFT in draft & published NFT, change mentioned in this https://github.com/Pylons-tech/pylons/issues/1404

### CODE REVIEW

File: easel/lib/screens/creator_hub/widgets/draft_list_tile.dart [hunk 1]
```diff
@@ -108,74 +109,84 @@ class _DraftListTileState extends State<DraftListTile> {
...
-      child: Padding(
-        padding: EdgeInsets.symmetric(horizontal: 15.w, vertical: 15.h),
-        child: Row(
-          children: [
-            buildAssetView(),
-            SizedBox(
-              width: 10.w,
+    return InkWell(
+      onTap: () {
+        widget.viewModel.startPublishingFlowAgain(startPublishingFlowAgainPressed: startPublishingFlowAgainPressed);
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/draft_list_tile.dart:120]: Why not use this      
```
  onTap: () {
        startPublishingFlowAgainPressed();
      },
```

File: easel/lib/screens/creator_hub/widgets/draft_list_tile.dart [hunk 2]
```diff
@@ -108,74 +109,84 @@ class _DraftListTileState extends State<DraftListTile> {
...
+      },
+      child: DecoratedBox(
+        decoration: BoxDecoration(
+          color: Colors.white,
+          boxShadow: [
+            BoxShadow(
+              color: Colors.black.withOpacity(0.3),
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/draft_list_tile.dart:127]: Should be from EaselTheme.

File: easel/lib/screens/creator_hub/widgets/draft_list_tile.dart [hunk 3]
```diff
@@ -108,74 +109,84 @@ class _DraftListTileState extends State<DraftListTile> {
...
-            buildAssetView(),
-            SizedBox(
-              width: 10.w,
+    return InkWell(
+      onTap: () {
+        widget.viewModel.startPublishingFlowAgain(startPublishingFlowAgainPressed: startPublishingFlowAgainPressed);
+      },
+      child: DecoratedBox(
+        decoration: BoxDecoration(
+          color: Colors.white,
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/draft_list_tile.dart:124]: Should be from EaselTheme.

File: easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart [hunk 4]
```diff
@@ -39,30 +54,26 @@ class NftGridViewItem extends StatelessWidget {
             key: const Key(kGridViewTileNFTKey),
             onTap: () {
               if (context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft) {
-                final DraftsBottomSheet draftsBottomSheet = DraftsBottomSheet(
-                  buildContext: context,
-                  nft: nft,
-                );
-                draftsBottomSheet.show();
+                context.read<CreatorHubViewModel>().startPublishingFlowAgain(startPublishingFlowAgainPressed: startPublishingFlowAgainPressed);
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:57]: Why not this 
```
 if (context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft) {
                startPublishingFlowAgainPressed();
                return;
              }
```

File: easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart [hunk 5]
```diff
@@ -19,15 +20,29 @@ import 'package:shimmer_animation/shimmer_animation.dart';
 
 import '../../../main.dart';
 
-class NftGridViewItem extends StatelessWidget {
+class NftGridViewItem extends StatefulWidget {
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:23]: Whats the reason for converting it to stateful?

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart [hunk 6]
```diff
@@ -30,85 +33,98 @@ class NFTsListTile extends StatelessWidget {
...
-                ),
-                assetType: publishedNFT.assetType.toAssetTypeEnum(),
-              ),
-            ),
-            SizedBox(
-              width: 10.w,
+    return InkWell(
+      key: const Key(kNftListTileKey),
+      onTap: () {
+        context.read<CreatorHubViewModel>().openOwnerView(openOwnerViewPressed: openOwnerView);
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:47]: Why not use `openOwnerView()`

File: easel/test/widget/nfts_list_tile_test.dart [hunk 7]
```diff
@@ -87,6 +91,34 @@ void main() {
           expect(banner, findsNothing);
         },
       );
+
+      testWidgets(
+        "can user tap on whole nft tile",
```
Comment (kjawadDeveloper2) [inline easel/test/widget/nfts_list_tile_test.dart:96]: Not a good approach . Use this. 

```

      testWidgets(
        "when the user press nft tile then owner view should open",
        (tester) async {
          await tester.setScreenSize();
          await tester.testAppForWidgetTesting(
            Scaffold(
              body: ChangeNotifierProvider(
                create: (ctx) => GetIt.I.get<CreatorHubViewModel>(),
                builder: (context, _) {
                  return NFTsListTile(
                    publishedNFT: MOCK_NFT,
                  );
                },
              ),
            ),
          );

          await tester.pump();
          final nftTile = find.byKey(const Key(kNftListTileKey));
          await tester.tap(nftTile);
          verify(viewModel.openOwnerView(openOwnerViewPressed: anyNamed("openOwnerViewPressed"))).called(1);
        },
      );
```

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]

**Comment by kjawadDeveloper2** [comment]
Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-11-29 at 19 07 43
Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-11-29 at 19 08 18

Since you are here kindly also resolve issue with these two. 
The publish text is in lower case. 
Second the icons in the bottom sheet are very small.

File: easel/lib/screens/creator_hub/widgets/draft_list_tile.dart (hunk 1 above)
Comment (ahmadrns) [reply easel/lib/screens/creator_hub/widgets/draft_list_tile.dart:120]: We need to write test case for this that's why need to implement this function in viewModel. Modified `startPublishingFlowAgain` function

File: easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart (hunk 4 above)
Comment (ahmadrns) [reply easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:57]: We need to write test case for this that's why need to implement this function in viewModel. Modified `startPublishingFlowAgain` function

File: easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart (hunk 5 above)
Comment (ahmadrns) [reply easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:23]: Used for `startPublishingFlowAgain` need to get context but not handling this in viewModel itself so reverting back to stateless

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart (hunk 6 above)
Comment (ahmadrns) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:47]: We need to write test case for this that's why need to implement this function in viewModel. Modified `openOwnerView` function

**Pull request closed without merging**